### PR TITLE
include token address in skip monitoring log

### PR DESCRIPTION
**Pull request by agatsoh**
## Description

Fixes:

Please, describe what this PR does in detail:
- The logs for skipping monitoring request dont specify for which token they are doing it. It could be helpful to know the token address.

### CODE REVIEW

**Comment by Dominik1999** [comment]
can you fix the linting here?
you can either run `make black` in the terminal or simply delete the space in 
`token_address= to_checksum_address(WETH_TOKEN_ADDRESS)` to 
`token_address=to_checksum_address(WETH_TOKEN_ADDRESS)`

File: raiden/services.py [hunk 1]
```diff
@@ -133,15 +134,19 @@ def update_monitoring_service_from_balance_proof(
             if channel_balance < MIN_MONITORING_AMOUNT_DAI:
                 log.warning(
                     message.format(
-                        channel_balance=channel_balance, min_balance=MIN_MONITORING_AMOUNT_DAI
+                        channel_balance=channel_balance,
+                        min_balance=MIN_MONITORING_AMOUNT_DAI,
+                        token_address=to_checksum_address(DAI_TOKEN_ADDRESS)
```
Comment (Dominik1999) [inline raiden/services.py:139]:
```suggestion
                        token_address=to_checksum_address(DAI_TOKEN_ADDRESS),
```

File: raiden/services.py [hunk 2]
```diff
@@ -133,15 +134,19 @@ def update_monitoring_service_from_balance_proof(
...
                 )
                 return
         if channel_state.canonical_identifier.token_network_address == weth_token_network_address:
             if channel_balance < MIN_MONITORING_AMOUNT_WETH:
                 log.warning(
                     message.format(
-                        channel_balance=channel_balance, min_balance=MIN_MONITORING_AMOUNT_WETH
+                        channel_balance=channel_balance,
+                        min_balance=MIN_MONITORING_AMOUNT_WETH,
+                        token_address=to_checksum_address(WETH_TOKEN_ADDRESS)
```
Comment (Dominik1999) [inline raiden/services.py:149]:
```suggestion
                        token_address=to_checksum_address(WETH_TOKEN_ADDRESS),
```

File: raiden/services.py [hunk 3]
```diff
@@ -113,7 +113,8 @@ def update_monitoring_service_from_balance_proof(
             "Skipping update to Monitoring service. "
             "Your channel balance {channel_balance} is less than "
             "the required minimum balance of {min_balance} "
-            "that you have set before sending the MonitorRequest"
+            "that you have set before sending the MonitorRequest,"
+            " token address {token_address}"
```
Comment (Dominik1999) [inline raiden/services.py:117]: Can you make this a whole sentence? Like

"... before sending the MonitorRequest, " 
"in the Token Network {token_address}."

File: raiden/services.py (hunk 1 above)
Comment (Dominik1999) [reply raiden/services.py:139]: you need to add a comma such that the linting check works

**Review by palango — APPROVED** [review]

**Pull request merged**